packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -649,6 +653,9 @@ const NavigationMenuContent = React.forwardRef<
   const commonProps = {
     value: itemContext.value,
     triggerRef: itemContext.triggerRef,
+    focusProxyRef: itemContext.focusProxyRef,
+    onContentFocusOutside: itemContext.onContentFocusOutside,
+    onRootContentClose: itemContext.onRootContentClose,
```

Comment:
Since I've moved the tab handling into the common `impl` I decided that passing the methods verbatim as props was clearer than trying to split logic across both the `impl` and the `itemContext` methods.